sdk/eventgrid/azure-eventgrid/CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # Release History
 
-## 4.11.1 (Unreleased)
+## 4.12.0b1 (Unreleased)
 
 ### Features Added
 
+- Added a new EventGridClient that supports publish, receive, acknowledge, release, and reject operations for CloudEvents.
+- Added new models to support the new operations on EventGridClient.
```

Comment:
Personally, I would add which models here and the names of the operations as well, but probably not a big deal:
```suggestion
- Added a new EventGridClient with the following methods: `publish_cloud_events`, `receive_cloud_events`, `acknowledge_cloud_events`, `release_cloud_events`, and `reject_cloud_events`.
 - Added the following models to support the new operations: `azure.eventgrid.Model1`, `azure.eventgrid.Model2`
```